docs/reference/commandline/context_inspect.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "context inspect"
+description: "The context inspect command description and usage"
+keywords: "context, inspect"
+---
+
+<!-- This file is maintained within the docker/cli GitHub
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+inspect
+# context import
```

Comment:
ITYM `inspect` not `import` and the preceding line looks like it shouldn't be there?